config/locales/en/shared.yml

```diff
@@ -213,6 +213,11 @@ en:
         deductions:
           dependants_allowance: Dependants allowance
           disregarded_state_benefits: Income from benefits excluded from calculation
+          partner_allowance: Partner Allowance
```

Comment:
nit: following the existing content case types, should `Partner Allowance` be `Partner allowance`